ci-operator/step-registry/ipi/conf/manual-creds/remove-unnecessary-creds/ipi-conf-manual-creds-remove-unnecessary-creds-commands.sh

```diff
@@ -63,6 +67,9 @@ case ${BASELINE_CAPABILITY_SET} in
 "v4.12")
   base_operators="${v412}"
   ;;
+"v4.13")
+  base_operators="${v413}"
```

Comment:
How about renaming the var name to `enabled_operators` for easy reading.

Reply:
updated.